src/Serilog/Capturing/PropertyValueConverter.cs

```diff
@@ -169,6 +185,61 @@ LogEventPropertyValue CreatePropertyValue(object? value, Destructuring destructu
         return new ScalarValue(value.ToString() ?? "");
     }
 
+    Destructuring? GetCachedFallbackDestructuring(
+#if !NET5_0
```

Comment:
.NET 5 had a bug that it was missing the `DynamicallyAccessedMemberTypes.Interfaces` item. But `type.GetInterfaces()` was also not using it, so here is no problem not adding it. Any other .NET version (Framework and Core) has it and would result in Roslyn analyzer issues if left away.